game/src/graphic/hud/EventResponsiveDialogue.java

```diff
@@ -0,0 +1,131 @@
+package graphic.hud;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Input;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.utils.Align;
+import controller.ScreenController;
+import tools.Constants;
+
+/** Predefined key event calls up dialogue with corresponding formatting and functionality */
+public class EventResponsiveDialogue<T extends Actor> extends ScreenController<T> {
+
+    /** Possible key events for dialogue call */
+    public enum AuthorisedAccessKeys {
```

Comment:
WTF?
Wofür ist das?
Was soll das denn? Warum musst du denn wissen welcher Knopf gedrückt wurde? 

Ich will doch nur einen Text auf den Hud anzeigen, warum klammerst du dich so sehr daran wo das aufgerufen wird und wo der text her kommt? Das kann dir doch echt egal sein....

Wenn für dich wichtig ist, wo und warum das Aufgerufen wird, dann erklärt mir das bitte. Vielleicht überseh ich ja etwas.

Reply:
Das war doch deine Aufgabenstellung
Spiel pausieren
msg in der GUI anzeigen
ok Button anzeigen
Wenn ok gedrückt wird -> Gui entfernen, spiel fortsetzen.
Das wurde soweit erfüllt.
Durch den Tastendruck wird das Spiel auf jegliche Art pausiert (ich weiß nicht in welchen Fällen bzw. unter welchen Umständen und nach welchen Aktivitäten der Dialog das Spiel pausieren soll). Die Strukturierung durch die Buttons kann der Dialog an verschiede Situationen angepasst werden. 
Soll ich alles Rückgängig machen und nur die Funktion showInfoText drin lassen?

Reply:
Bezüglich der anderen Sachen die du noch unten aufgeführt hast:
Warum ist es für dich wichtig, wo dieses showInfoText aufgerufen wird?
=> das wollte ich einfach zu testzwecken wissen, falls noch was erweitert werden sollte.

Warum schreibst du umständlich ein eigenes Paus Konstrukt, wenn es in Game doch schon eine Funktion gibt um alles zu pausieren?
=> Den Dialog kann man nicht ein und ausblenden wie das Label was ihr vorher auf dem Bildschirm ausgegeben habt. 
Nach jedem schließen des Dialoges. Das Ist auch der Grund weshalb ich die Pausierung etwas umbauen musste. 

Warum ist der PR immer noch nicht nach https://github.com/Programmiermethoden/Dungeon/wiki/Workflow gestaltet?
=> Branch werde ich in "Implementation_of_Info_GUI" umtaufen. Ja bei inteliJ muss ich demnächst vermehrt auf die Struktur der Commit Massege achten.

Warum ist der PR kein Draft PR, wenn das noch Work in Progress ist?
=> da du gestern auf meine Frage nicht geantwortet hast ob ihr noch etwas zu der Kollision haben wolltet, habe ich es so hochgeladen (wenn keine Aktion durch Kollision gewünscht ist, dann ist es auch nicht wirklich halbfertig).

Warum werden Konversationen in den Review Kommentaren nicht zuende geführt? Ich hab dich schonmal gefragt warum das "wo" des Aufrufst für dich so wichtig ist und keine Antwort bekommen.
=> ich werde ab jetzt pingelig darauf achten zum jeden Kommentar in der Review eine konkrete Antwort zu schreiben

Schreibe mir bitte konkret, was ich aus dem von mir generierten Code entfernen oder ändern soll.
Es wäre vielleicht hilfreich, wenn ihr mir sagen könntet wie das Spiel insgesamt geplant ist.